pycbc/events/eventmgr_cython.pyx

```diff
@@ -28,3 +28,20 @@ def findchirp_cluster_over_window_cython\
             indices[j] = i
             curr_ind = i
     return j
+
+@boundscheck(False)
+@wraparound(False)
+@cdivision(True)
+def coincbuffer_numgreater(
+    float[:] cbuffer,
+    int length,
+    float value
+):
+    cdef:
+        int idx, count
+
+    count = 0
+    for idx in range(length):
+        if cbuffer[idx] > value:
+            count += 1
```

Comment:
Maybe this is even (slightly?) faster like so, because you avoid a branch?
```
        count += cbuffer[idx] > value
```

Reply:
(I have no idea what kind of asm code Cython would generate in both cases though).

Reply:
This resulted in the function being about 160% faster than before this change ... I didn't check the machine level code, but it does result in different C source code.

I also realised that the test suite wasn't checking IFAR, so I've expanded the test suite to test these details as well.